### ISSUE TITLE
[BUG] - Fatal Database Error

### ISSUE THREAD

**Issue by Crip1** [akasolace/HO#772]
**Describe the bug**
I get the message,

Fatal database error.   Exiting HO!
You should restore the db-folder from backup or delete that folder

**To Reproduce**

1. Start HO 3.0
2. I get error message above
3. Click on "OK"
4. Restore database window comes up
5. Selecting and of the files shown,  *2020-10-28.zip, *2020-10-30.zip, or *2020-11-4.zip and clicking on Restore
6.   Nothing happens, HO Shuts down

**Screenshots**
Was unable to capture a screenshot

**Platform information:**
 - OS: Windows
 - Version (including build number)

**Additional context**
My father's team still works on HO on the same computer.

Tried to attach the zip files, but they are larger than 10MB (about 150MB), so Github won't let me upload them

**Comment by Crip1** [comment]
I deleted the .db folder, restored one of the .zip folder to a backup directory, copied the contents of the backup directory to the .db folder.     Same problem.     There was a .lck and .log file that I couldn't delete using windows file manager, which was saying Java was using them.     I got rid of the .lck and .log file another way, but still the same problem when I tried to run HO.

**Comment by akasolace** [comment]
hi @Crip1,
you can use wetransfer service to share large documents. Paste here the downloable link of the zipped database.

**Comment by Crip1** [comment]
Here's the we transfer link to the 3 zip folders

https://we.tl/t-pjtxiBa93O

And here's the link to the files currently in that folder (though they may have been based on what I extracted from one of the .zip folders last night).     

<<<I've been trying to upload the files themselves in wetransfer, but it keeps hanging up in the wetransfer processs.    I will keep trying, but meanwhile, I will just send this comment>>>

Yes, 2 users on the installation running 2 teams.    My files are under the db folder, my father's are under the db2 folder.     Everything is working fine for his team.

Only unusual things might have been a windows crash while HO! was open, and our ISP was extremely slow yesterday (pre-3pm Eastern in US)   Access to HT front page was intermittent, and and login attempt to HT failed (timed out probably).

**Comment by Crip1** [comment]
Turned out I didn't completely remove the database.lck and database.log files.  

Although they had disappeared from my file directory, the program I was using to remove them asked for a restart of the computer, which I didn't do.     This morning, the database.lck and database.log files were back in the db folder.      I removed them again, and this time, I did a hard re-boot of the computer.     Then, I started HO! and it worked.

Apologies for being a nuisance.    Ticket can be closed.
